pennylane/interfaces/tf.py

```diff
@@ -32,11 +32,13 @@
     from tensorflow import Variable  # pylint: disable=unused-import,ungrouped-imports
 
 
-def to_tf(qnode):
+def to_tf(qnode, dtype=None):
     """Function that accepts a :class:`~.QNode`, and returns a TensorFlow eager-execution-compatible QNode.
 
     Args:
         qnode (~pennylane.qnode.QNode): a PennyLane QNode
+        dtype (tf.DType): target output type of QNode; uses default output type of QNode if not
+            specified
```

Comment:
Slight subtlety, but not quite accurate. Instead, 'uses the TensorFlow equivalent of the QNode output type'. I.e., `float` becomes `tf.float32`, `np.float64` becomes `tf.float64`, etc.

The logic for how TensorFlow determines the equivalent type comes from [`tf.as_dtype()`](https://www.tensorflow.org/api_docs/python/tf/dtypes/as_dtype).